assets/nuclear-grammar.ne

```diff
@@ -63,13 +63,16 @@ Process prescripts.
 Prescripts are used a lot and are in a standard form
 */
 const getMassAndAtomicNumber = (prescript) => {
-    const regex = /\^{(?<mass>[0-9]+)}_{(?<atomic>-?[0-9]+)}/;
+    const regex = /\^{(?<mass>[0-9]+|())}_{(?<atomic>-?[0-9]+|())}/;
```

Comment:
This is certainly overly-pedantic so feel free to ignore, but your changes make a new capturing group to match the empty string, for which the mass / atomic groups are wrappers around. Since you only refer to the named groups it's okay, but it does mean `matches` will be in a slightly weird format (returns 4 matches) for anyone changing this in future.

There'd be nothing wrong with just making those groups non-capturing (`/\^{(?<mass>[0-9]+|(?:))}_{(?<atomic>-?[0-9]+|(?:))}/`), but I think this is a little cleaner:
```suggestion
    const regex = /\^{(?<mass>(?:[0-9]+)?)}_{(?<atomic>(?:-?[0-9]+)?)}/;
```